recipe/meta.yaml

```diff
@@ -18,22 +18,12 @@ build:
   number: 1
```

Comment:
Looks good but could you bump the build number to `2`?

Reply:
sure